server/api/api.go

```diff
@@ -4174,3 +4177,42 @@ func jsonBytesResponse(w http.ResponseWriter, code int, json []byte) {
 	w.WriteHeader(code)
 	_, _ = w.Write(json)
 }
+
+func (a *API) handleCloudLimits(w http.ResponseWriter, r *http.Request) {
+	// swagger:operation GET /api/v1/limits cloudLimits
```

Comment:
After the endpoint migration to `v2` this should be:

```suggestion
	// swagger:operation GET /limits cloudLimits
```